### ISSUE TITLE
Reply drafts language should default to original status language

### ISSUE THREAD

**Issue by niklaswolf** [elk-zone/elk#1609]
I would love Elk to behave like the mastodon default Web UI. When replying to an english toot, the language is automatically set to english. When replying to a german toot, it's set to german... in addition to a language preference for new toots of course.

_Originally posted by @sariash in https://github.com/elk-zone/elk/issues/1196#issuecomment-1383571520_
            
I think this is a separate issue I can work on.
Basically the language selection of reply drafts should default to the language of the original status.

**Comment by edimitchel** [comment]
This could be answered by using a configuration from mastodon itself which could be done with the Preference API usage: https://github.com/elk-zone/elk/issues/774.

I already start something on : see https://github.com/elk-zone/elk/pull/1200

**Comment by niklaswolf** [comment]
@edimitchel I see this on top and two different features. You can set a writing language preference, but you also can not. And I think many users actually engage in more than one language, so the writing language preference makes not too much sense for them. You wouldn't want to reply to a English post in German for example.

**Comment by edimitchel** [comment]
Ho yes sorry, I red the description too quickly.
I see your change and I think it makes sense!
